docs/how-to/add-the-nginx-route-relation.md

```diff
@@ -42,12 +42,12 @@ charmcraft pack
 ```
 This will build the charm inside an LXC container for you. The output will tell you the location of the built charm. For example, `my-charm_ubuntu-22.04-amd64.charm`.
```

Comment:
Nitpicky, but I think the tone of the documentation here might be a little to friendly.
```suggestion
This will build the charm inside an LXC container. The output is the location of the built charm. For example, `my-charm_ubuntu-22.04-amd64.charm`.
```